drivers/i2c-bitbang/i2c.py

```diff
@@ -0,0 +1,128 @@
+import sys
+import utime
+from machine import Pin
+
+
+class I2C:
+
+    def __init__(self, sda, scl):
+        sda.init(Pin.IN)
+        scl.init(Pin.IN)
+        sda(0)
+        scl(0)
```

Comment:
LOL. Indeed, funny glitch. That was straightforward translated of metaprogramming C++ code, and those args were template params. Of course, that needs to be fixed.